scout/constants/variant_tags.py

```diff
@@ -295,6 +295,24 @@
 }
 
 MANUAL_RANK_OPTIONS = {
+    12: {
+        "label": "URF",
+        "name": "Uncertain Risk Factor",
+        "description": "Uncertain risk allele - uncertain evidence for a small risk increase",
+        "label_class": "default",
+    },
+    11: {
+        "label": "LRF",
+        "name": "Likely Risk Factor",
+        "description": "Likely risk allele - some evidence for a small risk increase",
+        "label_class": "default",
+    },
+    10: {
+        "label": "RF",
+        "name": "Risk Factor",
+        "description": "Established risk allele - strong evidence for a small risk increase",
+        "label_class": "default",
```

Comment:
Perhaps here I would also do something about the order in which these terms are shown in the select.

<img width="238" alt="image" src="https://user-images.githubusercontent.com/28093618/212838017-ba2e4ab0-06a7-48a2-acd4-ae5509280a18.png">

 Risk factor tags are not tags used a lot (otherwise they would already be there) so perhaps shown them on the bottom or something? To do so one could convert MANUAL_RANK_OPTIONS to an OrderedDict

Reply:
Good call!
![Screenshot 2023-01-31 at 12 08 17](https://user-images.githubusercontent.com/758570/215744244-9bfb8bde-7594-4307-a9da-cb840e305ccf.png)